ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -658,15 +658,37 @@ tests:
       ASSISTED_CONFIG: |
         DEPLOY_TARGET=onprem
     workflow: baremetalds-assisted
-- as: e2e-metal-assisted-multi-version-suite
+- as: e2e-metal-assisted-4.6
   optional: true
-  run_if_changed: ^(data/.*)$
```

Comment:
usually we did this to trigger all tests on OCP bump PRs
how do you feel about keeping it?

Reply:
specifically for 4.6 we don't bump the version anymore, so maybe it can stay this way